### DISCUSSION TITLE
"Can't reach database server at" on docker and planetscale

### DISCUSSION THREAD

**Discussion by stewart-lore** [prisma/prisma#13420] [Q&A] [answered]
Hello I'm running prisma client in a docker container using a planetscale mysql database. I'm able to connect to the database just fine when running the application locally. However when I run the same app with the same database connection string in a docker container I get this--

```
Can't reach database server at `<planetscale>.us-west-2.psdb.cloud`:`3306`

Please make sure your database server is running at `<planetscale>.us-west-2.psdb.cloud`:`3306`.
```

In the application I also run a dig dns check on the database hostname and am able to get the ip address so I know it's not a networking issue with docker.

Does anyone have experience running prisma v3.14.0 on docker using node:14 as the base image?

**Comment by nurul3101** [comment, marked as answer]
Hey @stewart-prysm 👋 
Do you have any sample repository through which I could try to reproduce it?

**Reply by vladi-stevanovic** [reply]
👋 Hello @stewart-prysm  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Reply by stewart-lore** [reply]
I never found the root cause, but i belive the problem was from using the `node:14` docker container. Prisma client works for me when using `node:18-alpine`

```FROM node:18-alpine as base
WORKDIR /app
COPY . .

RUN yarn install
RUN yarn build

CMD ["yarn", "start"]```
Here's a very simple docker file with an app using prisma.

**Reply by vladi-stevanovic** [reply]
Thank you for sharing @stewart-prysm !

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
